Argo/Types/Decoded/Monad.swift

```diff
@@ -13,6 +13,21 @@ public func >>- <T, U>(x: Decoded<T>, @noescape f: T -> Decoded<U>) -> Decoded<U
   return x.flatMap(f)
 }
 
+/**
+  flatMap a function over a `Decoded` value (left associative)
+
+  - If the value is a failing case (`.TypeMismatch`, `.MissingKey`), the function will not be evaluated and this will return the value
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 134 characters (line_length)